### DISCUSSION TITLE
Is there a native undo function for Eto.Forms.RichTextArea?

### DISCUSSION THREAD

**Discussion by ghost** [picoe/Eto#2551] [Q&A] [answered]
Hello! I was curious whether or not there was some sort of native undo function for the `Eto.Forms.RichTextArea` control. I've searched for hours trying to find a solution, and I haven't come across any that have worked. Any help would be greatly appreciated!

**Comment by cwensley** [comment, marked as answer]
Not currently, sorry.  It would hopefully not be to difficult to add assuming each platform has some form of undo method.

**Reply by ghost** [reply]
Oh, I see. Thank you so much for your input! May you give me some pointers as to how I would go about implementing something like this into my project? If so, thanks again!
